config/default-amo.js

```diff
@@ -24,9 +24,24 @@ module.exports = {
   },
   enableTrailingSlashesMiddleware: true,
   fxaConfig: 'amo',
+
+  // This needs to be kept in sync with addons-server's SUPPORTED_NONLOCALES
```

Comment:
I think it would help to add a sentence about what the map is for -- something simple like "These URLs do not require a locale prefix."